src/app/games/hexodia/HexodiaTutorial.ts

```diff
@@ -0,0 +1,101 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { HexodiaState } from './HexodiaState';
+import { HexodiaMove } from './HexodiaMove';
+import { Coord } from 'src/app/jscaip/Coord';
+import { HexodiaConfig, HexodiaRules } from './HexodiaRules';
+import { TutorialStepMessage } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepMessage';
+import { FourStatePiece } from 'src/app/jscaip/FourStatePiece';
+import { MGPOptional } from '@everyboard/lib';
+
+const _: FourStatePiece = FourStatePiece.EMPTY;
+const O: FourStatePiece = FourStatePiece.ZERO;
+const X: FourStatePiece = FourStatePiece.ONE;
+const N: FourStatePiece = FourStatePiece.UNREACHABLE;
+const defaultConfig: MGPOptional<HexodiaConfig> = HexodiaRules.get().getDefaultRulesConfig();
+const initialState: HexodiaState = HexodiaRules.get().getInitialState(defaultConfig);
+
+export class HexodiaTutorial extends Tutorial {
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Initial board and object of the game`,
+            $localize`Hexagonal Connection is played on an hexagonal board, the aim is to be the first to align 6 of yours pieces.`,
```

Comment:
Hexodia*
. The aim is to be [...]
ou mieux:
. The object of the game is to be [...]

the first to align 6 pieces, pas "6 of yours pieces" ("le but est d'être le premier à aligner 6 de vos pièces" ça serait bizarre, si on joue à deux et que je t'explique les règles ça veut dire que le but est que j'aligne 6 de tes pièces)